praw/__init__.py

```diff
@@ -1122,14 +1122,16 @@ def search(self, query, subreddit=None, sort=None, syntax=None,
         :param period: The time period of the results.
 
         The additional parameters are passed directly into
-        :meth:`.get_content`. Note: the `url` and `param` parameters cannot be
-        altered.
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
 
         See https://www.reddit.com/wiki/search for more information on how to
         build a search query.
 
         """
         params = {'q': query}
+        if 'params' in kwargs:
```

Comment:
What other params other than query might one want to pass? Could we add those as arguments to search?